src/components/tabs/tabs.pw.tsx

```diff
@@ -0,0 +1,822 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+
+import { TabProps, TabsProps } from ".";
+import {
+  tabList,
+  tabById,
+  tabContentById,
+  tabTitleById,
+} from "../../../playwright/components/tabs";
+import {
+  TabsComponent,
+  TabsComponentValidations,
+  TabsComponentValidationsUnregistering,
+  TabsValidationOverride,
+  TabsInSidebar,
+} from "./components.test-pw";
+import {
+  WithAdditionalTitleSiblings,
+  WithAdditionalTitleSiblingsSizeLarge,
+  WithCustomLayout,
+  WithStringValidationsSummarised,
+  Responsive,
+} from "./tabs.stories";
+
+import {
+  getDataElementByValue,
+  tooltipPreview,
+} from "../../../playwright/components";
+import { ICON, PILL_PREVIEW } from "../../../playwright/components/locators";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  checkAccessibility,
+  assertCssValueIsApproximately,
+  checkGoldenOutline,
+  getStyle,
+} from "../../../playwright/support/helper";
+import { HooksConfig } from "../../../playwright";
+
+const validationType = ["error", "warning", "info"] as const;
```

Comment:
**suggestion(non-blocking):** this could be plural as it's an array of the types of validations. 

```suggestion
const validationTypes = ["error", "warning", "info"] as const;
```